cli/src/semgrep/commands/ci.py

```diff
@@ -323,9 +326,18 @@ def ci(
             is_full_scan = metadata.merge_base_ref is None
             engine = EngineType.OSS
             if scan_handler and scan_handler.deepsemgrep:
-                engine = (
-                    EngineType.PRO_INTERFILE if is_full_scan else EngineType.PRO_LANG
-                )
+                if is_full_scan:
+                    if pro_intrafile:
```

Comment:
A bit concerned that the flags don't mean the same as for `scan`, e.g. `--pro-intrafile --pro` in `scan` would be the same as `--pro` but here it would be the same as just `--pro-intrafile`.

Reply:
Yeah, I'm concerned about this too. Personally, I would like them to just be equivalent and override the toggle, but I don't have strong feelings.

Reply:
I think it's better to override the toggle too.